### PR TITLE
ci: cache eslint

### LINKED ISSUE #14184: eslintが遅いので結果をキャッシュしたい
eslintが遅い。特にフロントエンドは毎回8分ぐらいかかる。
https://madogiwa0124.hatenablog.com/entry/2022/08/07/112051 を参考にしてキャッシュしたい

### PR DESCRIPTION
**Pull request by KisaragiEffective**
## What

close #14184 

## Why

同上

## Checklist
- [x] Read the [contribution guide](https://github.com/misskey-dev/misskey/blob/develop/CONTRIBUTING.md)
- [ ] Test working in a local environment
- [ ] (If needed) Add story of storybook
- [ ] (If needed) Update CHANGELOG.md
- [ ] (If possible) Add tests

### CODE REVIEW

**Comment by KisaragiEffective** [comment]
blocked by #14205

**Comment by KisaragiEffective** [comment]
> The workflow is not valid. .github/workflows/lint.yml (Line: 69, Col: 23): Unrecognized named-value: 'eslint-cache-version'. Located at position 1 within expression: eslint-cache-version

File: .github/workflows/lint.yml [hunk 1]
```diff
@@ -59,7 +61,14 @@ jobs:
...
     - run: corepack enable
     - run: pnpm i --frozen-lockfile
-    - run: pnpm --filter ${{ matrix.workspace }} run eslint
+    - name: Restore eslint cache
+      uses: actions/cache@v4.0.2
+      with:
+        path: node_modules/.cache/eslint
+        key: eslint-${{ env.eslint-cache-version }}-${{ hashFiles('/pnpm-lock.yaml') }}-${{ github.ref_name }}-${{ github.sha }}
+        restore-keys: |
+          eslint-${{ eslint-cache-version }}-${{ hashFiles('/pnpm-lock.yaml') }}-
```
Comment (KisaragiEffective) [inline .github/workflows/lint.yml]:
```suggestion
          eslint-${{ env.eslint-cache-version }}-${{ hashFiles('/pnpm-lock.yaml') }}-
```

File: packages/backend/src/const.ts [hunk 2]
```diff
@@ -3,6 +3,7 @@
  * SPDX-License-Identifier: AGPL-3.0-only
  */
 
+// dummy
```
Comment (kakkokari-gtyih) [inline packages/backend/src/const.ts:6]: ？

Comment (KisaragiEffective) [reply packages/backend/src/const.ts:6]: 確かに要らない (シンタックスエラーが原因だったので)

**Comment by syuilo** [comment]
👍

**Pull request merged**